.github/workflows/vib-publish.yaml

```diff
@@ -0,0 +1,114 @@
+      - id: get-chart
+        name: 'Get modified charts'
+        run: |
+          cd charts
+          files_changed="$(git show --pretty="" --name-only)"
+          # Adding || true to avoid "Process exited with code 1" errors
+          charts_dirs_changed="$(echo "$files_changed" | xargs dirname | grep -o "bitnami/[^/]*" | sort | uniq || true)"
+          # Using grep -c as a better alternative to wc -l when dealing with empty strings."
+          num_charts_changed="$(echo "$charts_dirs_changed" | grep -c "bitnami" || true)"
+          num_version_bumps="$(echo "$files_changed" | grep Chart.yaml | xargs git show | grep -c "+version" || true)"
+
+          if [[ "$num_charts_changed" -ne "$num_version_bumps" ]]; then
+            # Changes done in charts but version not bumped -> ERROR
+            echo "::set-output name=error::Detected changes in charts without version bump in Chart.yaml.\nCharts changed: ${num_charts_changed}\n${charts_dirs_changed}\nVersion bumps detected: ${num_version_bumps}"
+            echo "::set-output name=result::fail"
+          elif [[ "$num_charts_changed" -eq "1" ]]; then
+            # Changes done in only one chart -> OK
+            chart_name=$(echo "$charts_dirs_changed" | sed "s|bitnami/||g")
+            echo "::set-output name=chart::${chart_name}"
+            echo "::set-output name=result::ok"
+          else
+            # Changes done in more than chart -> FAIL
+            echo -e "::set-output name=error::Changes detected in more than one chart directory:\n${charts_dirs_changed}\nThe publish process will be stopped. Please create different commits for each chart."
+            echo "::set-output name=result::fail"
+          fi
+      - id: show-error
+        name: 'Show error'
+        if: ${{ steps.get-chart.outputs.result == 'fail' }}
+        uses: actions/github-script@v3
+        with:
+          script: |
+              core.setFailed('${{ steps.get-chart.outputs.error }}')
+
+  vib-publish:
+    runs-on: ubuntu-latest
+    needs: get-chart
+    if: ${{ needs.get-chart.outputs.result == 'ok' }}
+    environment: vmware-image-builder
+    name: Publish
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout Repository
+        with:
+          path: charts
+      - uses: vmware-labs/vmware-image-builder-action@main
+        name: Verify and publish ${{ needs.get-chart.outputs.chart }}
+        with:
+          pipeline: ${{ needs.get-chart.outputs.chart }}/vib-publish.json
+          config: charts/.vib/
+        env:
+          VIB_ENV_TARGET_PLATFORM: 7ddab896-2e4e-4d58-a501-f79897eba3a0
+          VIB_ENV_ALTERNATIVE_TARGET_PLATFORM: 91d398a2-25c4-4cda-8732-75a3cfc179a1
+          VIB_ENV_S3_URL: ${{ secrets.S3_URL }}/bitnami
+          VIB_ENV_S3_USERNAME: ${{ secrets.S3_USERNAME }}
+          VIB_ENV_S3_PASSWORD: ${{ secrets.S3_PASSWORD }}
+          VIB_ENV_S3_ROLE: ${{ secrets.S3_ROLE }}
```

Comment:
I can't see that these secrets are currently defined. Could you please make sure they are before merging?